src/20250930_Multi_LaunchGHOOnPlasmaSetACIAsEmissionsManagerForRewards/AaveV3Plasma_LaunchGHOOnPlasmaSetACIAsEmissionsManagerForRewards_20250930_Part2.sol

```diff
@@ -0,0 +1,280 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'openzeppelin-contracts/contracts/token/ERC20/IERC20.sol';
+import {IACLManager} from 'aave-address-book/AaveV3.sol';
+import {SafeERC20} from 'openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol';
+import {AaveV3Plasma, AaveV3PlasmaAssets} from 'aave-address-book/AaveV3Plasma.sol';
+import {AaveV3PayloadPlasma} from 'aave-helpers/src/v3-config-engine/AaveV3PayloadPlasma.sol';
+import {GhoPlasma} from 'aave-address-book/GhoPlasma.sol';
+import {MiscPlasma} from 'aave-address-book/MiscPlasma.sol';
+import {GovernanceV3Plasma} from 'aave-address-book/GovernanceV3Plasma.sol';
+import {EngineFlags} from 'aave-v3-origin/contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-origin/contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {CCIPChainSelectors} from '../helpers/gho-launch/constants/CCIPChainSelectors.sol';
+import {IUpgradeableBurnMintTokenPool, IRateLimiter} from 'src/interfaces/ccip/IUpgradeableBurnMintTokenPool.sol';
+import {IEmissionManager} from 'aave-v3-origin/contracts/rewards/interfaces/IEmissionManager.sol';
+
+import {IGhoBucketSteward} from 'src/interfaces/IGhoBucketSteward.sol';
+import {IGhoToken} from 'src/interfaces/IGhoToken.sol';
+import {IGsm} from 'src/interfaces/IGsm.sol';
+import {IGsmRegistry} from 'src/interfaces/IGsmRegistry.sol';
+import {IAaveCLRobotOperator} from 'src/interfaces/IAaveCLRobotOperator.sol';
+
+interface IGhoReserve {
+  function addEntity(address entity) external;
+  function setLimit(address entity, uint256 limit) external;
+  function totalEntities() external view returns (uint256);
+}
+
+/**
+ * @title Add GHO and deploy GSM on Plasma.
+ * @author @TokenLogic
+ * - Snapshot: https://snapshot.box/#/s:aavedao.eth/proposal/0xeb3572580924976867073ad9c8012cb9e52093c76dafebd7d3aebf318f2576fb
+ * - Discussion: https://governance.aave.com/t/arfc-launch-gho-on-plasma-set-aci-as-emissions-manager-for-rewards/22994/6
+ */
+contract AaveV3Plasma_LaunchGHOOnPlasmaSetACIAsEmissionsManagerForRewards_20250930_Part2 is
+  AaveV3PayloadPlasma
+{
+  using SafeERC20 for IERC20;
+
+  uint128 public constant DEFAULT_RATE_LIMITER_CAPACITY = 1_500_000e18;
+  uint128 public constant DEFAULT_RATE_LIMITER_RATE = 300e18;
+
+  // https://plasmascan.to/address/0xac140648435d03f784879cd789130F22Ef588Fcd
+  address public constant GHO_LM_ADMIN = 0xac140648435d03f784879cd789130F22Ef588Fcd;
+
+  uint256 public constant GHO_SEED_AMOUNT = 10e18;
+
+  // GhoReserve
+  // https://plasmascan.to/address/0xBAdA742e7Ff54595F9049eeF1Cc5AaF4364988B9
+  address public constant GHO_RESERVE = 0xBAdA742e7Ff54595F9049eeF1Cc5AaF4364988B9;
+  uint256 public constant BRIDGED_AMOUNT = 50_000_000 ether;
+
+  // Capacities
```

Comment:
```suggestion
  // GSM
```
i d put gsm address, fee strat and usdt capacity below
everything preffixed with `GSM_USDT` or `GSM_AUSDT`